homeassistant/components/onvif/__init__.py

```diff
@@ -33,8 +32,6 @@
 )
 from .device import ONVIFDevice
 
-CONFIG_SCHEMA = vol.Schema({DOMAIN: vol.Schema({})}, extra=vol.ALLOW_EXTRA)
```

Comment:
This actually does an import into the config flow, should it be marked deprecated instead?

Reply:
Nevermind, that is from platform level.